video/out/hwdec/hwdec_drmprime.c

```diff
@@ -56,7 +56,7 @@ const static dmabuf_interop_init interop_inits[] = {
 #if HAVE_DMABUF_INTEROP_PL
     dmabuf_interop_pl_init,
 #endif
-#if HAVE_DMABUF_INTEROP_WL
+#if HAVE_DMABUF_WAYLAND
```

Comment:
This should also be part of the previous commit (i.e. so there's no diff here in the commit that removes vaapi-wayland).